segmentation_models_pytorch/decoders/pan/model.py

```diff
@@ -58,6 +58,8 @@ def __init__(
         activation: Optional[Union[str, callable]] = None,
         upsampling: int = 4,
         aux_params: Optional[dict] = None,
+        encoder_kwargs: Optional[dict] = None,
+        encoder_depth: int = 5,
```

Comment:
well, this is probably out of the scope of this PR. Did you test that PAN works with a depth other than 5?

Reply:
let me try it

Reply:
tbh, I have changed this by mistake, although this change should not hurt. Do you want me to remove it?

Reply:
Yes, could you remove this code pls?